modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -696,6 +706,400 @@ properties:
+
+                      The `max_backoff_time` configuration property specifies the time-period (in minutes) during which Sync Gateway will attempt to reconnect lost or unreachable `remote` targets.
+
+                      **Behavior**
+
+                      On disconnection Sync Gateway performs an exponential backoff up to the specified value, after which it will attempt to reconnect indefinitely every `max_backoff_time` minutes.
+
+                      If a zero value is specified then Sync Gateway will do an exponential backoff up to an interval of five minutes before stopping the replication.
+
+                      **Constraints**
+
+                      The value defaults to five minutes for replications created prior to version 2.8.
+
+                  perf_tuning_params:
+                    type: array
+                    description: |2
+                      The `perf_tuning_params` are not available in this release.
+
+                    items:
+                      type: string
+
+                  purge_on_removal:
+                    type: boolean
+                    default: False
+                    description: |2
+
+                      **About**
+
+                      The optional `purge_on_removal` configuration property determines, per replication, whether the removal of a `channel` triggers a purge.
+
+                      **Options**
+                      - `true` or `false`
+                      - Default = false -- document removals are ignored by receiving end
+
+                      **Behavior**
+
+                      If `purge_on_removal=false`, then the removal of channels is ignored (not purged) by the receiving end.
+
+                      **Constraints**
+
+                      Replications created prior to version 2.8 *must* be run with `purge_on_removal=false`.
+
+                  query_params:
+                    type: array
+                    default: None - no query_params are used
+                    description: |2
+                      **About**
+
+                      The **`query_params`** configuration property defines a set of key/value pairs used in the query string of the replication.
+
+                      **Behavior**
+
+                      This property works in conjunction with [filters](./../refer/config-properties.html#databases-this_db-replications-filter) and [channels](./../refer/config-properties.html#databases-this_db-replications-channels) to provide routing.
+
+                      **Using**
+
+                      You can use `query_params`'s' *channels* function to *pull* from a specific set of `channels`.
+                      To do so, you would also need to set the `filter` to  `sync_gateway/bychannels`.
+
+                      For example :
+
+                      ```json
+                        "filter":"sync_gateway/bychannel",
+                        "query_params": {
+                          "channels":["channel.thisChannel"]
```

Comment:
The channel name `channel.thisChannel` is a bit confusing - it looks like the `channel.` prefix is required.  I think just `thisChannel` would be preferable.